src/app/garfile/responsibleperson/index.njk

```diff
@@ -26,147 +26,149 @@
     <div class="govuk-grid-column-two-thirds">
       <form id="page-form" action="/garfile/responsibleperson" autocomplete="off" class="form-group"
       method="post" role="form">
-        <span class="govuk-caption-xl">{{__('section_count', 5, 9)}}</span>
-        <h1 class="govuk-heading-xl">{{__('heading_responsible_person')}}</h1>
-        <p class="govuk-body-l">{{__('responsible_person_caption')}} </p>
+        <span class="govuk-caption-xl">{{ __('section_count', 5, 9) }}</span>
+        <h1 class="govuk-heading-xl">{{ __('heading_responsible_person') }}</h1>
+        <p class="govuk-body-l">{{ __('responsible_person_caption') }} </p>
         <div class="govuk-form-group {{ m.form_group_class('responsiblePerson', errors) }}">
           <div class="govuk-form-group {{ m.form_group_class('responsibleGivenName', errors) }}">
-            <label class="govuk-label govuk-label--m" for="responsibleGivenName">{{__('field_given_name')}}</label>
-            {{ m.hint('firstName', __('field_given_name_hint')) }}
```

Comment:
This hint was part of [FAIR 490](https://collaboration.homeoffice.gov.uk/jira/browse/FAIR-490).

Question: Is there a reason why it was removed?

Reply:
I probably didn't merge your changes in correctly.  My mistake.

Reply:
No problem.